Added missed step on the Scheduled Changes page (#1496)
magento/merchdocs: src/catalog/product-scheduled-changes.md

1. On the top bar click <span class="btn">Remove from Update</span>.

> 1. In the opened modal, select **Delete the Update** and click **Done** button

Reviewer: For user docs, we don't use "modal". When a response is required, it is a "dialog".

```suggestion
1. In the dialog, select **Delete the Update** and click **Done**.
```

> {:.bs-callout-info}
> The product will be removed from the update and all scheduled changes will be lost.

Reviewer: ```suggestion
   {:.bs-callout-info}
   The product is removed from the update and all scheduled changes are lost.
```
